apps/cli/src/node/driver/k8s.ts

```diff
@@ -0,0 +1,64 @@
+import { execa } from "execa";
+import semver from "semver";
+
+import { NodeDriver, NodeStatus } from "./index.js";
+import { DApp } from "../database/index.js";
+
+export class K8sDriver implements NodeDriver {
+    constructor() {}
+
+    async init(): Promise<void> {
+        // check helm installed version
+        const { stdout: helmVersion } = await execa("helm", [
+            "version",
+            "--template",
+            "{{.Version}}",
+        ]);
+
+        const minimumVersion = ">=3.0.0";
+        if (!semver.satisfies(helmVersion, minimumVersion)) {
+            throw new Error(`Helm version must be ${minimumVersion}`);
+        }
+    }
+
+    async start(dapp: DApp, location: string): Promise<NodeStatus> {
+        const name = dapp.address.substring(2, 8).toLocaleLowerCase();
```

Comment:
```suggestion
        const name = dapp.address.substring(1, 7).toLocaleLowerCase();
```

a Service name can't start with a number